t/ui/18-tests-details.t

```diff
@@ -169,6 +198,23 @@ $t->element_count_is('.tab-pane.active', 1, 'only one tab visible at the same ti
 my $href_to_isosize = $t->tx->res->dom->at('.component a[href*=installer_timezone]')->{href};
 $t->get_ok($baseurl . ($href_to_isosize =~ s@^/@@r))->status_is(200);
 
+subtest 'Developer instructions' => sub {
+    # TODO: Actively check for the job state
+
+    my $element_search;
+    # find* will croak/warn, make sure we trap that
+    # https://metacpan.org/pod/Selenium::Remote::Driver#find_element
+    ok($driver->get($baseurl . 'tests/100001'));
+    my $warning = warning { $element_search = $driver->find_element_by_class_name("developer-instructions") };
```

Comment:
Why do you wrap in a mysterious `warning` block? If the element might not exist, I usually use `@elements = $driver->find_elements('.developer-instructions');` and do a check like `is(scalar @elements, 1, 'at least one element present');`.

Reply:
@Martchus We're using Test::Warnings in our tests modules, which tries to make sure that there are no warnings in the test code/application code so when calling `$driver->find_element_by_class` if no elements are found, the Selenium carps emiting a `warn` that is then catched by the Test::Warnings causing the `had_no_warnings` test to fail :)

Reply:
this is still no proper way to test. Either the element needs to be there or not. Testing if it's there or not in a test is just nonsense

Reply:
@coolo [indeed](https://github.com/os-autoinst/openQA/pull/1671/commits/7de64642f3cbb4ac8853d56e8bced7d7286e7eaa#diff-7b77b47860f0cd1f6dfd7aef3c62efbeR178) VNC port and worker's hostname are the elements that should be tested...